ui/src/app/tasks/task-definitions/task-definitions.component.ts

```diff
@@ -18,6 +18,13 @@ export class TaskDefinitionsComponent implements OnInit {
   busy: Subscription;
   taskDefinitionToDestroy: TaskDefinition;
 
+  // null indicates that order is not selected
+  // thus we simply show double arrow.
+  // for sorting toggles, we just switch
+  // and first sort with true which equals desc
+  definitionNameSort: boolean = null;
+  definitionSort: boolean = null;
```

Comment:
I wonder if we should make that somehow part of Pageable - see Spring Data's `org.springframework.data.domain.PageRequest` and also have it be part of the service in order to have some stateful behavior.